src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -200,83 +178,63 @@ public boolean isPrivilegeCheckboxInPermissionDivChecked(int instructorIndex, St
         return checkbox.isSelected();
     }
     
-    public WebElement courseLevelPanel(int instrNum) {
-        String permissionDivId = "tunePermissionsDivForInstructor" + instrNum;
-        return browser.driver.findElement(By.id(permissionDivId)).findElement(By.cssSelector("div.form-group>div>div.panel"));
-    }
-    
     public void clickCourseLevelPrivilegesLink(int instrNum, int linkNum) {
         WebElement coursePanel = this.courseLevelPanel(instrNum);
-        coursePanel.findElements(By.cssSelector("input[type=checkbox]")).get(linkNum - 1).click();
-    }
-    
-    public WebElement addSectionLevelPrivilegesLink(int instrNum) {
-        String idStr = "addSectionLevelForInstructor" + instrNum;
-        
-        return browser.driver.findElement(By.id(idStr));
+        coursePanel.findElements(By.cssSelector("input[type='checkbox']")).get(linkNum - 1).click();
     }
     
     public void clickAddSectionLevelPrivilegesLink(int instrNum) {
         this.addSectionLevelPrivilegesLink(instrNum).click();
     }
     
-    private WebElement sectionCheckBoxInSectionLevel(int instrNum, int sectionLevelNum, int sectionNum) {
-        String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " input[name=sectiongroup" + sectionLevelNum + "section" + sectionNum + "]";
-        return browser.driver.findElement(By.cssSelector(cssSelector));
+    public void clickSectionCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex, int sectionNum) {
+        this.sectionSelectionCheckBox(instrNum, sectionLevelIndex - 1, sectionNum - 1).click();
```

Comment:
Missed this out previously: should rename `clickSectionCheckBoxInSectionLevel()` to `clickSectionSelectionCheckBox` for consistency